src/services/s3/schema.graphql

```diff
@@ -65,6 +65,17 @@ type awsS3AclGrant
   permission: String @search(by: [hash])
 }
 
+type awsS3ServerSideEncryptionConfiguration
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+  id: String! @id
+  sseAlgorithm: String @search(by: [hash])
+  kmsMasterKeyID: String @search(by: [hash])
```

Comment:
Do we want to add a kms connection to S3 based on this field?

Reply:
@tyler-dunkel Not really, at first I left it in case it was needed for some rule but finally I didn't use it, only `sseAlgorithm` is needed. I can remove it in that case, unless you prefer me to add the connection, what do you think?

Reply:
Lets keep it and add the connection, more data couldnt hurt! lol

Reply:
ok great!